cmake/modules/grpc.cmake

```diff
@@ -48,47 +50,49 @@ else()
 
 		# fixme(leogr): this workaround is required to inject the missing deps (built by gRCP cmakefiles)
 		# into target_link_libraries later
+		# note: the list below is manually generated starting from the output of pkg-config --libs grpc++
```

Comment:
Any chance it could be sorted alphabetically? How is this updated?

Also, probably missing something, but:
```
root@44806f25c37b:/# pkg-config --libs grpc++
-lgrpc++
```